docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 11
+title: Local Datastore (FactDB)
+---
+
+# PDP Local Datastore (FactDB)
```

Comment:
1. You don't need the #1 here, it will take it from the `title:` in the header.
2. Please convert the next 2 paragraphs to one TL;DR text that explains what is the outcome of reading the page, and not the objective of the page.
3. The TL;DR should be designed as the header of the page (not title, header)

See structure example here: https://github.com/permitio/docs/blob/master/docs/modeling/mesa-verde.mdx

Reply:
Shortened the header